content/shmem_team_get_options.tex

```diff
@@ -0,0 +1,44 @@
+\apisummary{
+    shmem\_team\_get\_options returns the options flags describing a given team
+}
+
+\begin{apidefinition}
+
+\begin{Csynopsis}
+long @\FuncDecl{shmem\_team\_get\_options}@(shmem_team_t team);
+\end{Csynopsis}
+
+\begin{apiarguments}
+\apiargument{IN}{team}{A valid SHMEM team handle.}
+\end{apiarguments}
+
+\apidescription{
+\FUNC{shmem\_team\_get\_options} returns a long unsigned value containing
+all of the options which describe the given team. Options are requested when
+new teams are created in the various \FUNC{shmem\_team\_split\_*} functions.
+Whichever of the requested options are applied to the team by the library
+implementation will be returned by \FUNC{shmem\_team\_get\_options}.
+
+All processes in the team will get back the same value for the team options.
+
+Error checking will be done to ensure a valid team handle is provided.
+All errors are considered fatal and will result in the job aborting
+with an informative error message.
```

Comment:
Do we really want to abort on error for `shmem_team_get_options`? If an implementation is required to check the validity of the team handle, I think this call should return -1 when provided an invalid team handle.

Reply:
Agreed. This is better for get_options. This also seems better as an option for team_translate.